cinema/views.py

```diff
@@ -1,17 +1,23 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.utils import extend_schema, OpenApiParameter
 from rest_framework import viewsets, mixins, status
-from rest_framework.authentication import TokenAuthentication
 from rest_framework.decorators import action
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from rest_framework.response import Response
-from rest_framework.viewsets import GenericViewSet, ReadOnlyModelViewSet
-
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession, Order
+from rest_framework.viewsets import GenericViewSet
```

Comment:
The import statement for `GenericViewSet` is correct, but ensure that it is used consistently throughout the file for defining viewsets.